scripts/build.sh

```diff
@@ -45,11 +45,16 @@ mkdir -p $FM_CFG_DIR
 mkdir -p $FM_ELECTRS_DIR
 
 # Copy configs to data directories
-cp misc/test/bitcoin.conf $FM_BTC_DIR
 cp misc/test/lnd.conf $FM_LND_DIR
 cp misc/test/lightningd.conf $FM_CLN_DIR/config
 cp misc/test/electrs.toml $FM_ELECTRS_DIR
 
+# Defining a variable which contains the project root
+FM_SRC_DIR=`$PWD`
+
+# Exporting it to be used externally
+export FM_SRC_DIR=$FM_SRC_DIR
```

Comment:
You can just export [SRC_DIR](https://github.com/fedimint/fedimint/blob/master/scripts/build.sh#L27) variable as `export $FM_SRC_DIR=$SRC_DIR`

Reply:
It is giving lint error, suggesting to do it separately

Reply:
oh there src dir above, rename that `FM_SRC_DIR` and export it

Reply:
@dhruv-1001 Then should I directly export `SRC_DIR`? Or make new variable `FM_SRC_DIR`

Reply:
As @Maan2003 suggested, you can rename `SRC_DIR` to `FM_SRC_DIR` just to follow the convention. And make sure to change `SRC_DIR` to `FM_SRC_DIR` everywhere in `build.sh`